gradle-license-plugin/src/main/kotlin/com/jaredsburrows/license/internal/report/HtmlReport.kt

```diff
@@ -66,6 +66,13 @@ class HtmlReport(private val projects: List<Model>) : Report {
       (projectsMap[key] as MutableList).add(project)
     }
 
+    val sortedProjectsList =
+      projectsMap.entries.map { (key, projects) ->
+        Pair(key, projects.sortedWith(compareBy(String.CASE_INSENSITIVE_ORDER) { it.name }))
+      }.sortedWith(compareBy(String.CASE_INSENSITIVE_ORDER) { (key, license) -> key })
```

Comment:
Possible to remove unused variable `license` or use `_`?

Reply:
Fixed 7477790302bb7007560e7a97c5f265e1058c1da8